src/x/sync/pooled_worker_pool.go

```diff
@@ -62,13 +64,15 @@ func NewPooledWorkerPool(size int, opts PooledWorkerPoolOptions) (PooledWorkerPo
 	}
 
 	return &pooledWorkerPool{
-		numRoutinesAtomic:     0,
-		numRoutinesGauge:      opts.InstrumentOptions().MetricsScope().Gauge("num-routines"),
-		growOnDemand:          opts.GrowOnDemand(),
-		workChs:               workChs,
-		numShards:             numShards,
-		killWorkerProbability: opts.KillWorkerProbability(),
-		nowFn: opts.NowFn(),
+		numRoutinesAtomic:       0,
+		numActiveRoutinesAtomic: 0,
+		numRoutinesGauge:        opts.InstrumentOptions().MetricsScope().Gauge("num-routines"),
+		numActiveRoutinesGauge:  opts.InstrumentOptions().MetricsScope().Gauge("num-active-routines"),
```

Comment:
Nit: what do you think about renaming `active-routines` to `working-routines`? Maybe a little clearer what is meant.